packages/core/src/main/kubectl/request-greatest-patch-version.injectable.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { TypedRegEx } from "typed-regex";
+import fetchInjectable from "../../common/fetch/fetch.injectable";
+import loggerInjectable from "../../common/logger.injectable";
+import { XMLParser } from "fast-xml-parser";
+
+export type RequestGreatestKubectlPatchVersion = (majorMinor: string) => Promise<string | undefined>;
+
+const expectedResponseForm = TypedRegEx("^v(?<version>\\d+\\.\\d+\\.\\d+)$");
+
+const requestGreatestKubectlPatchVersionInjectable = getInjectable({
+  id: "request-greatest-kubectl-patch-version",
+  instantiate: (di): RequestGreatestKubectlPatchVersion => {
+    const fetch = di.inject(fetchInjectable);
+    const logger = di.inject(loggerInjectable);
+
+    return async (majorMinor) => {
+      const response = await fetch(`https://dl.k8s.io/release/stable-${majorMinor}.txt`);
```

Comment:
Maybe we should extract fetching URL to some `*.injectable.ts` or common function?
Btw, is it only `xml`-response for their apis available? No plain `json`?

Reply:
Yeah only XML (for error messages). Normal (200) response is just plain text